demisto_sdk/commands/split_yml/extractor.py

```diff
@@ -220,7 +220,7 @@ def extract_code(self, code_file_path) -> int:
                 code_file.write("import demistomock as demisto\n")
             if common_server:
                 if lang_type == TYPE_PYTHON:
-                    code_file.write("from CommonServerPython import *\n")
+                    code_file.write("from CommonServerPython import *  # noqa: F401\n")
```

Comment:
dont we also need here `E402 lgtm [py/polluting-import]` ?

Reply:
e402 is something else. The py/polluting-import of lgtm we ignore globally at: https://github.com/demisto/content/blob/master/.lgtm.yml